blacktie/jatmibroker-xatmi/src/main/java/org/jboss/narayana/blacktie/jatmibroker/core/transport/OrbManagement.java

```diff
@@ -26,9 +26,9 @@
 public class OrbManagement implements Runnable {
     private static final Logger log = LogManager.getLogger(OrbManagement.class);
     private static final String CorbaOrbClassProp = "org.omg.CORBA.ORBClass";
-    private static final String CorbaOrbClassValue = "org.jacorb.orb.ORB";
+    private static final String CorbaOrbClassValue = "com.sun.corba.se.impl.orb.ORBImpl";
```

Comment:
Hi Mike,

I just changed to use the com.sun.corba.se.impl.orb.ORBImpl
@tomjenkinson does it make sense ?

Amos